c3/c3objs.py

```diff
@@ -294,7 +294,15 @@ def _set_value_extend(self, val) -> None:
         """Set the value of this quantity as tensorflow. If needed, limits will be extended."""
         min_val, max_val = self.get_limits()
         # Extra bounds included to not be directly at border due to differentiability
-        minmax = [val * 0.9, val * 1.1, min_val, max_val]
+        # val can be matrix valued
```

Comment:
Is there already a regression test that tries to force updates of values beyond the bounds and checks that it is correctly handled? Basically something that mocks what the Optimizer does in #195